mailbox/api/src/main/java/org/apache/james/mailbox/ApplicableFlagBuilder.java

```diff
@@ -0,0 +1,64 @@
+/****************************************************************
+ * Licensed to the Apache Software Foundation (ASF) under one   *
+ * or more contributor license agreements.  See the NOTICE file *
+ * distributed with this work for additional information        *
+ * regarding copyright ownership.  The ASF licenses this file   *
+ * to you under the Apache License, Version 2.0 (the            *
+ * "License"); you may not use this file except in compliance   *
+ * with the License.  You may obtain a copy of the License at   *
+ *                                                              *
+ *   http://www.apache.org/licenses/LICENSE-2.0                 *
+ *                                                              *
+ * Unless required by applicable law or agreed to in writing,   *
+ * software distributed under the License is distributed on an  *
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY       *
+ * KIND, either express or implied.  See the License for the    *
+ * specific language governing permissions and limitations      *
+ * under the License.                                           *
+ ****************************************************************/
+package org.apache.james.mailbox;
+
+import javax.mail.Flags;
+
+import com.google.common.annotations.VisibleForTesting;
+
+public class ApplicableFlagBuilder {
+
+    @VisibleForTesting
+    static final Flags DEFAULT_APPLICABLE_FLAGS = FlagsBuilder.builder().add(Flags.Flag.ANSWERED,
```

Comment:
I think it's make reading more difficult that `Flags.Flag.ANSWERED` is the only one to not be aligned with other. If you have time could you realign with the other